src/App/Infrastructure/Ui/Assets/scss/app/_helpers/_aspect-ratio.scss

```diff
@@ -0,0 +1,30 @@
+// This file is part of the Php DDD Standard project.
+//
+// Copyright (c) 2017 LIN3S <info@lin3s.com>
+//
+// For the full copyright and license information, please view the LICENSE
+// file that was distributed with this source code.
+//
+// @author Andrés Montejo <andres@lin3s.com>
+
+@mixin aspect-ratio($width, $height) {
```

Comment:
Should this be here? Seems a hack and can be extracted to a package and install it only in those case that is needed